<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="add_person">
      <property name="text">
       <string>Agregar técnico</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_person">
      <property name="text">
       <string>Eliminar técnico</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_boss">
      <property name="text">
       <string>Agregar jefe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_boss">
      <property name="text">
       <string>Eliminar jefe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
